.svglintrc.js

```diff
@@ -129,6 +131,34 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "icon-precision";
+
+            const iconPath = $.find("path").attr("d");
```

Comment:
Hey @service-paradis, the icons ignorance is missing.

```suggestion
            const iconPath = $.find("path").attr("d");
            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
              return;
            }
```

Reply:
Good catch! I was a little surprised this problem didn't show up with the Travis build but I figured out why. See my comments below.